src/pages/login/login.js

```diff
@@ -50,6 +54,22 @@ const Login = () => {
       .required(' ')
   });
 
+  useEffect(() => {
+    window.gapi.load('auth2', () => {
+      window.gapi.auth2.init({
+        client_id: process.env.REACT_APP_GOOGLE_CLIENT_ID
+      });
+    });
+  }, []);
+
+  const singIn = () => {
+    const googleAuth = window.gapi.auth2.getAuthInstance();
+    googleAuth.signIn().then((googleUser) => {
+      const id_token = googleUser.getAuthResponse().id_token;
```

Comment:
Don't use underscore in your `const` declaration

Reply:
Fixed